victor-nach/propertypro-lite
# #166692092 create landing page

-- pr description
#### What does this PR do?
create landing page

#### Description of Task to be completed?
- design landing page UI elements 

#### How should this be manually tested?
Clone this branch.
navigate to https://victor-nach.github.io/propertypro-lite/UI to see the landing page

#### What are the relevant pivotal tracker stories?
#166692092

#### Screenshots
![Screenshot from 2019-06-24 16-41-48](https://user-images.githubusercontent.com/46886694/60032740-1ce13b00-969f-11e9-85ca-1656a8bfdd94.png)
![Screenshot from 2019-06-24 16-42-31](https://user-images.githubusercontent.com/46886694/60032741-1d79d180-969f-11e9-8fc2-4fd0032208dc.png)
